### ISSUE TITLE
"Harvested" materials don't auto-stack like 3 yrs ago

### ISSUE THREAD

**Issue by lukipr0** [ProjectCeleste/Celeste_Server#223]
Back in 2014 materials could be picked up from the workshops and they would always automatically fill up their respective stacks (until full stack, then start a new stack).

Right now harvesting is more time-consuming than intended. Also you need to always make quite some space in inventory so you can harvest all at once. Once you did that, you still have to put together the materials to stacks, which really is prettttty unfortunate.

**Comment by DaantjuhNL** [comment]
Not a big issue, but still annoying

**Comment by jiffyette** [comment]
yes annoying i can pack up all my moes and sell 20 at one time 📦
